rbc/src/main/java/com/library/rbc/model/dto/ImageDto.java

```diff
@@ -0,0 +1,11 @@
+package com.library.rbc.model.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@AllArgsConstructor
+@Data
+public class ImageDto {
+  String imagePath;
```

Comment:
This field should be private, and you may need the `@NoArgsConstructor` annotation.